README.md

```diff
@@ -44,12 +44,34 @@ cargo build --release
 
 **Pura** can be used as a standalone container runtime like **runc** with the OCI compliant commands:
 
+Make this to clean the way:
+```sh
+set ID=example
+[sudo] unlink /tmp/pura/${ID}/init.sock
+[sudo] unlink /tmp/pura/${ID}/container.sock
 ```
+
+On another terminal:
+```sh
+[sudo] nc -vklU /tmp/${ID}.sock # on another terminal
+```
+
+On the config.json
+```sh
+...
+"root": {
+  "path": "/path/to/rootfs"
+}
+...
+```
+
+Now on pura context
+```sh
 cd target/release
-./pura create id123456789 --bundle /path/to/bundle
-./pura start id123456789
-./pura state id123456789
-./pura delete id123456789
+./pura create ${ID} --bundle /path/to/bundle --console-sock /tmp/${ID}.sock
```

Comment:
` --console-sock` option is required just in the case when `process.terminal` in the config.json is set to true. It's not required for running container.